nbterm/nbterm.py

```diff
@@ -14,12 +15,12 @@ def version_callback(value: bool):
 
 
 def main(
-    notebook_path: str = typer.Argument("", help="Path to the notebook."),
+    notebook_path: Path = typer.Argument(..., help="Path to the notebook."),
```

Comment:
Not quite sure what the intent was here - if the goal is to make this required, I think you need the `...` https://typer.tiangolo.com/tutorial/arguments/help/#add-a-help-text-for-a-cli-argument